apps/crn-server/test/integration/reminders.integration-test.ts

```diff
@@ -242,6 +251,253 @@ describe('Reminders', () => {
     });
   });
 
+  describe('Research Output Draft Reminder', () => {
+    let creatorId: string;
+    let notCreatorId: string;
+    let creatorUsername: string;
+    let teamId: string;
+    let fetchRemindersOptions: FetchRemindersOptions;
+
+    beforeEach(async () => {
+      const teamCreateDataObject = getTeamCreateDataObject();
+      teamCreateDataObject.applicationNumber = chance.name();
+      teamId = await teamDataProvider.create(teamCreateDataObject);
+
+      const userCreateDataObject2 = getUserInput(teamId);
+
+      creatorId = await userDataProvider.create(userCreateDataObject2);
+      let creator = await userDataProvider.fetchById(creatorId);
+      creatorUsername = creator?.firstName + ' ' + creator?.lastName;
+
+      const userCreateDataObject = getUserInput(teamId);
+      const userId1 = await userDataProvider.create(userCreateDataObject);
+      notCreatorId = userId1;
+
+      const timezone = 'Europe/London';
+      fetchRemindersOptions = { userId: userId1, timezone };
+    });
+
+    test('Should see the reminder when the research output was created recently and the user is associated with the team that owns it', async () => {
+      const researchOutputInput = getResearchOutputInputDraft(
+        teamId,
+        creatorId,
+      );
+      const researchOutputId = await researchOutputDataProvider.create(
+        researchOutputInput,
+        { publish: false },
+      );
+
+      const researchOutput = await researchOutputDataProvider.fetchById(
+        researchOutputId,
+      );
```

Comment:
Because this data is created inside the test you will want to move the `fetchById` inside the `retryable` call in order to make sure it is not subject to a race condition with the graphql cache.